website.qmd

```diff
@@ -0,0 +1,9 @@
+### Package website modes
+
+Epiverse-TRACE packages use the [`{pkgdown}` package](https://pkgdown.r-lib.org/) to generate websites. `{pkgdown}` provides [development modes](https://pkgdown.r-lib.org/reference/build_site.html#development-mode) (in the [`_pkgdown.yml` file](https://pkgdown.r-lib.org/articles/customise.html)) to generate different sites for the development and released versions of your package.
+
+When the package is not yet on CRAN, we use the `unreleased` development mode. In this case, we use a visual element to highlight the package development version in red. An example is the [website](https://epiverse-trace.github.io/superspreading/) for `{superspreading}` and its associated `_pkgdown` [configuration](https://github.com/epiverse-trace/superspreading/blob/main/_pkgdown.yml#L12-L13).
+
+After the package is accepted on CRAN, we switch to `auto` mode, which provides two versions of the website: `release` and `devel`. Here, an example is the `{cfr}`package [website](https://epiverse-trace.github.io/cfr/), which is configured [here](https://github.com/epiverse-trace/cfr/blob/main/_pkgdown.yml#L5-L6). We use this configuration for the CRAN packages because we expect most users to install the package from CRAN and so we direct them to the CRAN site associate with the installed release. The additional `devel` option here allows users to manually switch to it if they installed the development version from GitHub.
+
+We also provide text in package READMEs to direct users to the appropriate site. Typical text for the README might be: "The package has two websites: one for `[the stable release version on CRAN](URL)`, and another for `[the version in development](URL)`", where `URL` is the link to the respective site. See an example in the [`{EpiNow2}`] package [README](https://github.com/epiforecasts/EpiNow2/blob/main/README.Rmd#L125-L127).
```

Comment:
I would reword this from "We also provide" to something like "It is recommended to provide", as {EpiNow2} is not an Epiverse hosted package and I'm unaware of any Epiverse packages that state this in their READMEs (please point out packages that do this as it would be good to know).

Reply:
There's none but we need to start doing that so that users are not confused when they visit the website and it doesn't correspond to the CRAN version. I'll remove it and we can add it after a wider discussion.